kubectl/plan.sh

```diff
@@ -0,0 +1,60 @@
+pkg_name=kubectl
+pkg_origin=core
+pkg_license=('Apache 2.0')
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_version="1.6.3"
+pkg_source="https://storage.googleapis.com/kubernetes-release/release/v${pkg_version}/bin/linux/amd64/kubectl"
+pkg_shasum="2e9872fe2409deee257434a8c45976a4e1cc3f2e4e2a972fd85e86127a1f7cbd"
```

Comment:
I get `5ce1511888e3aae24f5e104af038a89b16fa7053b9c3d8a8e4afc23a4262c9f3` for this.